stellargraph/connector/neo4j/graph.py

```diff
@@ -0,0 +1,113 @@
+__all__ = ["Neo4jStellarGraph"]
+
+import numpy as np
+import scipy.sparse as sps
+from ...core.experimental import experimental
+
+
+@experimental(reason="the class is not fully tested and lacks documentation")
+class Neo4jStellarGraph:
+    def __init__(self, graph_db, is_directed=False):
+
+        self.graph_db = graph_db
+        self._is_directed = is_directed
+
+    def nodes(self):
+        # FIXME: don't assume stellargraphs "ID"s are in the Neo4j graph
+        # use id() instead
+        node_ids_query = f"""    
+            CALL apoc.cypher.run(
+                'MATCH (n)
+                RETURN n.ID as node_ids',
+                {{}}
+            ) YIELD value
+
+            RETURN collect(value.node_ids) as node_ids
+            """
+        result = self.graph_db.run(node_ids_query)
+
+        return result.data()[0]["node_ids"]
+
+    def node_features(self, node_ids):
+        feature_query = f"""
+            UNWIND $node_id_list AS node_id
+
+            // for each node id in every row, collect the random list of its neighbors.
+            CALL apoc.cypher.run(
+
+                'MATCH(cur_node) WHERE cur_node.ID = $node_id
+                RETURN cur_node.features as features',
+                {{node_id: node_id}}
+            ) YIELD value
+
+            RETURN collect(value.features) as features
```

Comment:
Why is `node_features` nominally looking at neighbours?

I think this might be something like:

```suggestion
            UNWIND $node_id_list AS node_id
            MATCH(node) WHERE node.ID = $node_id
            RETURN collect(node.features) as features
```

Alternatively, it could be something like:


```suggestion
            MATCH(node) WHERE node.ID IN $node_id_list
            RETURN value.features as features
```

In either case, we'll need to be careful about the orderings.